src/ucp/core/ucp_context.c

```diff
@@ -27,6 +27,16 @@
 
 #define UCP_RSC_CONFIG_ALL    "all"
 
+/* Seconds between keepalive rounds */
+#define UCP_CONTEXT_KEEPALIVE_TIMEOUT 1
+
+/* Maximal number of endpoints to check on every keepalive round */
+#define UCP_CONTEXT_KEEPALIVE_NUM_EPS 128
```

Comment:
UCP_CONTEXT_KEEPALIVE_DEFAULT_NUM_EPS

Reply:
renamed to UCP_CONTEXT_KEEPALIVE_DEFAULT_NUM_EPS